CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-tokenization ChangeLog
 
+## 7.1.0 - 2021-xx-xx
```

Comment:
I'm pretty sure this is a breaking change as support for the older keys is removed by this, right?

Reply:
Correct, updated.